code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -539,6 +539,52 @@
 	log_admin("[key_name(G)] was made a golem by [key_name(user)].")
 	qdel(src)
 
+//Ghost-to-Human rune for badmins
+
+/obj/effect/golemrune/humanrune		//Summons humans instead of golems - for admins exclusively
+	name = "Admin Rune"
+	desc = "A rune used by badmins to turn ghosts into humans. Oh boy."
+
+	New()
+		..()
+		SSobj.processing |= src
+
+/obj/effect/golemrune/humanrune/process()
```

Comment:
Do we really need our own process? Can't we piggy back off the `golemrune` process?